docs/site/Deploying-with-pm2-and-nginx.md

```diff
@@ -78,7 +78,7 @@ section of documentation for detailed instructions.
    };
    ```
 
-3. Add pm2 as a dependency to your projet using following command
+3. Add `pm2` as a dependency to your project using following command
```

Comment:
nitpick: `using following` -> `using the following`

Reply:
Done.